packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/CollectionItemScopeHandler/collectionItemIterationScopeHandler.ts

```diff
@@ -0,0 +1,24 @@
+import { testRegex } from "@cursorless/common";
+import type { TargetScope } from "../scope.types";
+import type { ComplexScopeType } from "../scopeHandler.types";
+import { separatorRegex } from "./getSeparatorOccurrences";
+
+export const collectionItemIterationScopeHandler: ComplexScopeType = {
+  type: "fallback",
+  scopeTypes: [
+    {
+      type: "surroundingPairInterior",
+      delimiter: "collectionBoundary",
+    },
+    {
+      type: "conditional",
+      scopeType: {
+        type: "line",
+      },
+      predicate: (scope: TargetScope) => {
+        const text = scope.editor.document.getText(scope.domain);
+        return testRegex(separatorRegex, text);
+      },
+    },
+  ],
```

Comment:
NTS: This is triggered for cases like having a comma separated list inside of a comment or prose without a bounding pair.

This does mean that you cannot iterate over lists that span multiple lines if they aren't in a surrounding pair of some sort (say in plaintext)